io.brackets.Brackets.appdata.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>io.brackets.Brackets.desktop</id>
+  <name>Brackets</name>
+  <developer_name>Adobe</developer_name>
+  <summary>An open source code editor for the web</summary>
+  <metadata_license>MIT</metadata_license>
+  <project_license>MIT</project_license>
+  <url type="homepage">http://brackets.io</url>
+  <description>
+    <p>Brackets is a modern open-source code editor for HTML, CSS and JavaScript that's built in HTML, CSS and JavaScript.</p>
+    <ul>
+	<li><strong>Tools shouldn't get in your way.</strong> Instead of cluttering up your coding environment with lots of panels and icons, 		the Quick Edit UI in Brackets puts context-specific code and tools inline.</li>
```

Comment:
I've not tested this, but I imagine `\<strong>` doesn't work?

Reply:
Probably none, but I'm not sure either.

Reply:
`<strong>` should work inside xml.